### PR TITLE
[MTM-47904] New SSO access mapping configuration enabled

### PR DESCRIPTION
**Pull request by Michal-Bak**
Documentation for 10.16

### CODE REVIEW

File: content/users-guide/administration-bundle/changing-settings.md [hunk 1]
```diff
@@ -330,11 +330,17 @@ In this case the following claim will match the condition:
...
+By default, dynamic access mapping assigns user roles, based on the access token, on every user login.
+This means, that it is not possible to change the user roles inside {{< product-c8y-iot >}} as these would be overridden on next user login.
+To change this behaviour, select one of the following radio button at the bottom of the **Access mapping** section. :
+* **Use dynamic access mapping only on user creation**
+  * When selected, dynamic access mapping will be used only when a new user logs in to fill in the initial roles. When a user already exists in {{< product-c8y-iot >}}, the roles will not be overridden nor updated.
+* **Roles selected in the rules above will be reassigned to a user on each log in and other ones will be unchanged**
+  * When selected, dynamic access mapping will be used on every login, but the roles not listed in the access mapping configuration will not be updated. Only the roles that are listed in the defined access mapping rules will be overwritten.
```
Comment (chbakar) [inline content/users-guide/administration-bundle/changing-settings.md:340]: The third option is not described

**Review by chbakar — APPROVED** [review]

File: content/users-guide/administration-bundle/changing-settings.md [hunk 2]
```diff
@@ -330,11 +330,16 @@ In this case the following claim will match the condition:
 
 As you can see, there is an option to verify if a value exists in a list via the "in" operator. Values can also be embedded in other objects. In this case a dot in the key implies looking into an embedded object.
 
-By default, dynamic access mapping assigns user roles, based on the access token, on every user login. This means, that it is not possible to change the user roles inside {{< product-c8y-iot >}} as these would be overridden on next user login. To change this behaviour, select the **Use dynamic access mapping only on user creation** checkbox at the bottom of the **Access mapping** section.
+By default, configuration for dynamic access mapping is selected: **Roles selected in the rules above will be reassigned to a user on each log in and other ones will be cleared** This means, that dynamic access mapping assigns user roles, based on the access token, on every user login, so it is not possible to change the user roles inside {{< product-c8y-iot >}} as these would be overridden on next user login.
```
Comment (rssag) [inline content/users-guide/administration-bundle/changing-settings.md:333]:
```suggestion
By default, the configuration for dynamic access mapping is selected: **Roles selected in the rules above will be reassigned to a user on each log in and other ones will be cleared**.
This means that dynamic access mapping assigns user roles, based on the access token, on every user login.
It is not possible to change the user roles inside {{< product-c8y-iot >}} as they would be overwritten on the next user login.
```

File: content/users-guide/administration-bundle/changing-settings.md [hunk 3]
```diff
@@ -330,11 +330,16 @@ In this case the following claim will match the condition:
...
+To change this behaviour, select one of the following radio button at the bottom of the **Access mapping** section. :
+* **Use dynamic access mapping only on user creation**
+  * When selected, dynamic access mapping will be used only when a new user logs in to fill in the initial roles. When a user already exists in {{< product-c8y-iot >}}, the roles will not be overridden nor updated.
+* **Roles selected in the rules above will be reassigned to a user on each log in and other ones will be unchanged**
+  * When selected, dynamic access mapping will be used on every login, but the roles not listed in the access mapping configuration will not be updated. Only the roles that are listed in the defined access mapping rules will be overwritten.
```
Comment (rssag) [inline content/users-guide/administration-bundle/changing-settings.md:338]:
```suggestion

* **Use dynamic access mapping only on user creation**

  When selected, dynamic access mapping will be used only when a new user logs in to fill in the initial roles. When a user already exists in {{< product-c8y-iot >}}, the roles will not be overwritten nor updated.
  
* **Roles selected in the rules above will be reassigned to a user on each log in and other ones will be unchanged**

  When selected, dynamic access mapping will be used on every login, but the roles not listed in the access mapping configuration will not be updated. Only the roles that are listed in the defined access mapping rules will be overwritten.
```

(blank lines and indentation are needed for the correct line breaks)

File: content/users-guide/administration-bundle/changing-settings.md [hunk 4]
```diff
@@ -330,11 +330,16 @@ In this case the following claim will match the condition:
...
 
 ![Custom access mapping](/images/users-guide/Administration/sso-custom-access-mapping-2.png)
 
-When selected, dynamic access mapping will be used only when a new user logs in to fill in the initial roles. When a user already exists in {{< product-c8y-iot >}}, the roles will not be overridden nor updated. Selecting this option also enables admins to edit roles of SSO users in the user management. For details, refer to  [Administration > Managing permissions](/users-guide/administration/#attach-global) in the *User guide*.
+Selecting one of the two options mentioned above will also enable admins to edit roles of SSO users in the user management. For details, refer to  [Administration > Managing permissions](/users-guide/administration/#attach-global) in the *User guide*.
```
Comment (rssag) [inline content/users-guide/administration-bundle/changing-settings.md:342]:
```suggestion
Selecting one of the two options mentioned above will also enable admins to edit roles of SSO users in the user management. For details, refer to [Administration > Managing permissions](/users-guide/administration/#attach-global) in the *User guide*.
```

**Review by rssag — CHANGES_REQUESTED** [review]

**Comment by Michal-Bak** [comment]
@rssag  Can you re-review?

**Review by rssag — APPROVED** [review]

**Pull request merged**